UI/Web/src/app/collections/_components/collection-detail/collection-detail.component.ts

```diff
@@ -225,6 +226,17 @@ export class CollectionDetailComponent implements OnInit, AfterContentChecked {
     });
   }
 
+  shouldRenderCollection(action: ActionItem<UserCollection>, entity: UserCollection, user: User) {
```

Comment:
These should be named shouldRenderCollectionAction to align with other code signatures